### ISSUE TITLE
Moldels and textures issues

### ISSUE THREAD

**Issue by jochuan** [Thutmose/Pokecube-Models#15]
i found some issues while i was looking in ther modfel reloader
Typhlosion back should be blue 
![2016-07-26_17 27 18](https://cloud.githubusercontent.com/assets/11268912/17155230/9fb3c844-535a-11e6-90b4-84111357362f.png)
Raikou should have black stripes
![2016-07-26_17 31 11](https://cloud.githubusercontent.com/assets/11268912/17155275/c417deb4-535a-11e6-8f8a-30f47805a18e.png)
![250px-243raikou](https://cloud.githubusercontent.com/assets/11268912/17155296/d9a94b8c-535a-11e6-879f-afe7c513323e.png)
Wurmple should have red spikes on the side too
![2016-07-26_17 32 18](https://cloud.githubusercontent.com/assets/11268912/17155374/35c22b82-535b-11e6-971b-60543648060e.png)
![250px-265wurmple](https://cloud.githubusercontent.com/assets/11268912/17155402/4cddac06-535b-11e6-8790-eec2e98f1c76.png)

**Comment by jochuan** [comment]
also raikou is missing the upper part of the muzzle in form of "x"

**Comment by Thutmose** [comment]
Here are the models you requested that I convert to obj:

[converted.zip](https://github.com/Thutmose/Pokecube-Models/files/465693/converted.zip)

**Comment by NelkGames** [comment]
I imported models in obj to the blender, but were in the wrong position, if you can put files already in blender I can pack faster.

**Comment by Thutmose** [comment]
Ok, I will try to get that done for you when I get back to my computer in about 4 hours.

**Comment by NelkGames** [comment]
I am unable to use the command / pokemake says I need permission to use, even with the cheats on and using the command block

**Comment by Thutmose** [comment]
[models.zip](https://github.com/Thutmose/Pokecube-Models/files/468280/models.zip)

Here they are as .blend files.

**Comment by Thutmose** [comment]
I have got the /pokemake issue fixed for next release, it seems I had messed something up and command blocks couldn't use it.

**Comment by NelkGames** [comment]
I play in version 1.9.4, so I'll wait out the update. :smile:

**Comment by NelkGames** [comment]
Could you get the models of pikachu and raichu for min? I exported them for their program but they were bugged. If you can send it in obj or fbx it will be great! Wanted to make a version of Christmas pikachu of Pokemon Go.

And I wanted to talk about a problem, the achievements of pokédex for me are failing, I can not pick up the items and the message appears several times.

**Comment by Thutmose** [comment]
I can't get the models for those two, they are hard coded in, so they are not in a format that I can export to put into blender

**Comment by NelkGames** [comment]
I got it, thanks.

Wanted to ask a question, will gyms be added someday?

**Comment by Thutmose** [comment]
yes, if you can make a gym that is smaller than 32x32x32, then I can get them added now

**Comment by NelkGames** [comment]
Amazing! I'll test some constructions and send it to you. What format does it need to be? Mcedit?

**Comment by Thutmose** [comment]
it uses the vanilla .nbt format, which can be exported by the structure block, that is why the limit of 32x32x32, as I think that is the largest size the structure block can export

**Comment by NelkGames** [comment]
I have some bugs in the mod, the mega evolution is failing and the pokedex is not giving the items of the achievements.
